non-critical-infra/modules/backup-share.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs, ... }:
+{
+  environment.systemPackages = [ pkgs.cifs-utils ];
+  fileSystems."/mnt/backup-share" = {
```

Comment:
Storage boxes support Borgbackup over SSH, which should be significantly more performant than over CIFS, fwiw.

Not sure if it really matters here since we're talking about a tiny amount of data anyway.

Reply:
Thanks, I used the login way I was given, but we'll change if we have to work with more data.